pmmlserver/rockcraft.yaml

```diff
@@ -34,35 +39,29 @@ parts:
     source-subdir: python
     source-tag: v0.10.0
     build-packages:
-    - wget
+    - python3.9
+    - python3-pip
+    - python3.9-distutils
     overlay-packages:
     - openjdk-11-jdk
+    - python3.9
```

Comment:
do you really need all this overlay package ? some can be on stage-packages no ?